### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by phoenixy1**
Improve windows instructions

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -131,6 +133,8 @@ $ make stop language=node
   This repo should generally work with active LTS versions of each language such as node >= 12,
   python >= 3.8, ruby >= 2.6, etc.
 - Your environment variables populated in `.env`
+- [npm](https://www.npmjs.com/get-npm)
+- If using Windows, a command line utility (such as Powershell or Cygwin) capable of running basic Unix shell commands
```
Comment (skylarmb) [inline README.md:137]: i dont believe powershell can run unix commands? you would need Cygwin or MinGW or gitbash or WSL etc.

File: README.md [hunk 2]
```diff
@@ -111,6 +108,11 @@ The quickstart backend is now running on http://localhost:8000 and frontend on h
 If you make changes to one of the server files such as `index.js`, `server.go`, etc, or to the
 `.env` file, simply run `make up language=node` again to rebuild and restart the container.
 
+If you experience a Docker connection error when running the command above, try the following:
+
+- Make sure Docker is running
+- Try running the command prefixed with `sudo`
```
Comment (skylarmb) [inline README.md:114]: this _should_ never be needed, as in if you need to do this you likely set up your development environment or docker incorrectly, but i guess its ok to call out the corner case

**Review by skylarmb — DISMISSED** [review]

File: README.md (hunk 1 above)
Comment (phoenixy1) [reply README.md:137]: powershell doesn't have full Unix shell support, but it does have built-in aliasing for simple commands (like ls, touch, etc.) which I think is good enough for running the basic scripts we have

Comment (phoenixy1) [inline README.md:137]:
```suggestion
- If using Windows, a command line utility capable of running basic Unix shell commands
```

**Review by skylarmb — APPROVED** [review]

**Pull request merged**